woocommerce/woocommerce-android
# Issue/5113 create quick order

Closes #5113 by creating the quick order from the quick order view, and navigating to the newly created order detail.
 
To test:

* Make sure "Quick order" is enabled in Settings > Beta features
* Go to the order list
* Tap the + icon
* Enter an amount
* Tap Done
* Ensure you're taken to the order detail
* Ensure order is created with the correct amount

![view](https://user-images.githubusercontent.com/3903757/141178280-1b2d5e43-0476-41c0-85d8-8d5ac23ed652.png)
